databricks/koalas/series.py

```diff
@@ -1748,6 +1748,72 @@ def reindex(self, index: Optional[Any] = None, fill_value: Optional[Any] = None,
             self.name
         )
 
+    def reindex_like(self: "Series", other: Union["Series", "DataFrame"]) -> "Series":
+        """
+        Return a Series with matching indices as other object.
+
+        Conform the object to the same index on all axes. Optional
+        filling logic, placing NaN in locations having no value
+        in the previous index.
+
+        Parameters
+        ----------
+        other : Series or DataFrame
+            Its row and column indices are used to define the new indices
+            of this object.
```

Comment:
Anyway, maybe pandas has `copy` parameter for `Series.reindex_like`, too ??

Even if it doesn't really do any meaningful work, how about adding it for compatibility with pandas?

Reply:
Yes we can, so we should also add it to `Series.reindex`.

Reply:
Yes pandas has `copy` parameter for `Series.reindex_like`. If we add it, we need to update `Series.reindex` too, which does not support `copy` parameter either.
Should I do the change ?

Reply:
Hmm... okay let's keep it as it is for now and discuss separately later.
Thanks for the opinion!

Reply:
Ok :)